### PR TITLE
Add babel-dynamic-import-webpack plugin to frontend-next

### PR DESCRIPTION
**Pull request by DenisOleksiuk**

### CODE REVIEW

File: frontend-next.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ module.exports = {
     'dynamic-import-node',
     ['babel-plugin-styled-components', {ssr: true, fileName: false, pure: true}],
     '@babel/plugin-proposal-optional-chaining',
```
Comment (vladgolubev) [inline frontend-next.js:9]: No need to add that

CleanShot 2021-07-13 at 18 15 58@2x

File: frontend-next.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ module.exports = {
...
+    'babel-plugin-dynamic-import-webpack',
```
Comment (slavik-chapelskyi) [inline frontend-next.js:10]: Why is this needed?
There is already `dynamic-import-node`

File: frontend-next.js (hunk 1 above)
Comment (DenisOleksiuk) [reply frontend-next.js:9]: but we haven't `presets: ['next/babel']` in frontend-next config?

File: frontend-next.js (hunk 2 above)
Comment (DenisOleksiuk) [reply frontend-next.js:10]: is it actually the same things?

Comment (slavik-chapelskyi) [reply frontend-next.js:10]: Ok the question remains opened.
Why it is needed?
`dynamic-import-node` is needed to have proper dynamic import in nextjs projects during ssr in dev mode.
From the documentation of `dynamic-import-webpack` it says this plugin is only needed for webpack v1, as v2 >= have implemented `import()` natively

https://www.npmjs.com/package/babel-plugin-dynamic-import-webpack
image
image

Comment (DenisOleksiuk) [reply frontend-next.js:10]: I wanted to add it to this config because it exists in .babelrc files in other projects like app-cpw, app-my-account, app-xxx and can be any other. But if this is not necessary, I close this PR and remove this rule in those projects.

Comment (slavik-chapelskyi) [reply frontend-next.js:10]: Please take a responsibility on this to check if this plugin is needed in the projects you have mentioned. 
If they will works without it, so no need to add plugin to babel config.
